src/pfe/file-watcher/server/src/utils/kubeutil.ts

```diff
@@ -185,9 +203,15 @@ export async function getApplicationContainerInfo(projectInfo: ProjectInfo, oper
  *
  * @returns Promise<any>
  */
-export async function isContainerActive(containerName: string): Promise<any> {
+export async function isContainerActive(containerName: string, projectInfo?: ProjectInfo): Promise<any> {
     try {
-        const releaseLabel = "release=" + containerName;
+        let releaseLabel = "release=" + containerName;
+        if (projectInfo.projectType == "odo") {
```

Comment:
Same here. We should delegate that to the extension code instead of hardcoding on this layer of the code.